docs/table/operations.rst

```diff
@@ -1088,6 +1088,24 @@ problem of "fuzzy" matching of key column values, where instead of an exact matc
 we require only an approximate match. This is supported using the ``join_funcs``
 argument.
 
+.. warning::
+
+   The coordinate and distance table joins discussed in this section are most
+   applicable in the case where the relevant entries in at least one of the
+   tables are all separated from one another by more than join distance. If this
```

Comment:
it should be twice the join distance. The point is that there should not be a single entry in the one table that matches two entries in the other.

And this is not just a characteristic of the distance joins. It is a characteristic of the underlying algorithm (as you say below). Even if a user provides a custom function, the join will only be guaranteed to be correct if there is at least one table with no entry matched to two entries in the other table.